bot_channel.go

```diff
@@ -9,5 +13,28 @@ import (
 // (see api.config.PrimaryChannel)
 func (app App) HandleChannelMessage(message discordgo.Message) error {
 	debug("[channel:HandleChannelMessage] %v", message)
+
+	if message.Content[:1] == "/" {
+		command := strings.Split(strings.ToLower(message.Content), " ")
+
+		switch command[0] {

```

Comment:
This could be lowercased, just the command symbol though, not the command contents in case future commands involve text.